### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by scottcreynolds**

### CODE REVIEW

**Comment by scottcreynolds** [comment]
@AnnJohn

File: README.md [hunk 1]
```diff
@@ -2,10 +2,28 @@
...
-  * Remind them that if they did everything correctly, they should be able to not make any changes to their view code
+1. Replace using `ProductSerializer` with using `to_json` to serialize
+   the `Product`. Only include the product `id`, `name`, `description`,
+`inventory` and `price` in the JSON response.
+2. Remove the old `products/:id/data` route and action and set
+   `products#show` to render JSON or HTML depending on the format of the
+request.
+3. Modify the `show.html.erb` page code to properly request JSON from
+   the `products/id` route.
+4. Make sure tests pass! Some will pass at the beginning. The trick is
```
Comment (pletcher) [inline README.md]: Good to note :)

File: README.md [hunk 2]
```diff
@@ -2,10 +2,28 @@
...
+   making sure they all *still* pass at the end!
+
+**Note:** If you do everything correctly, you won't have to change any
+of the code that constructs the `show` view, only the the route to
```
Comment (pletcher) [inline README.md]: :+1:

**Comment by pletcher** [comment]
LGTM

**Pull request merged**